static/styles/settings.css

```diff
@@ -175,13 +175,20 @@ h3,
     }
 }
 
+#deactivate_realm_button_container {
+    &.realm_deactivation_tooltip {
+        cursor: not-allowed;
+    }
+}
+
 #deactivate_account_container {
     &.only_organization_owner_tooltip {
         cursor: not-allowed;
     }
 }
 
-#user_deactivate_account_button {
+#user_deactivate_account_button,
+#deactivate_realm_button {
     &:disabled {
         pointer-events: none;
     }
```

Comment:
This is basically fine since there's only two of these, but I think the way that we'd ideally write this sort of code would be a have a `settings_deactivation_button` class, a `settings_deactivation_button_wrapper` class, a `settings_deactivation_tooltip` class, and we'd just have two instances of that component using those classes (and a unique ID for javascript).

Reply:
That's good feedback for me since the demo organization work will be disabling some additional settings buttons, and we might want to use a tooltip when that happens. So let me do a follow-up with that adjustment.